### DISCUSSION TITLE
Is this for dev's only?!

### DISCUSSION THREAD

**Discussion by terry9873** [dngray/ghacks-thunderbird-user.js#48] [Q&A]
I have moved away from Mac, now on Fedora and learning to use Thunderbird. I detest the way it tries to be a web browser, got shock of my life when I saw there were cookies in my EMAIL program?! 

A friend sent me a link to this 'github'. I like the sound of it, but wanted to ask:

1. Is it something an average user can install? (I barely understand how to navigate github, never mind find a way to install something mentioned therein!
2. Are there any drawbacks or problems caused in TB usage when this is installed? (I only use email and MAY use calendar as it has better reminder alerts than Fedora does)

Thanks, and thanks for making this.

**Comment by jacek-kubek** [comment]
There is a good, detailed [Wiki](https://github.com/HorlogeSkynet/thunderbird-user.js/wiki) for HorlogeSkynet/thunderbird-user.js. In the Wiki there is a link to [The Thunderbird Privacy Guide for Dummies!](https://12bytes.org/the-thunderbird-privacy-guide-for-dummies/). Both of them explain to you how to use `user.js` with Thunderbird.
Good luck :-)

**Comment by HorlogeSkynet** [comment]
Hello to both of you :wave:

> Thanks, and thanks for making this.

First of all, thanks for your message, it's appreciated.

> Is it something an average user can install? (I barely understand how to navigate github, never mind find a way to install something mentioned therein!

Yes, even if it is not trivial for non-IT people. Indeed this project wiki contains (or at lease _should_) required information. I advise you to browse "upstream" [Arkenfox wiki](https://github.com/arkenfox/user.js/wiki) too (focused on Firefox, but more recent/complete, it also gives a good background for these projects).

> Are there any drawbacks or problems caused in TB usage when this is installed? (I only use email and MAY use calendar as it has better reminder alerts than Fedora does)

Yes and no. You can look for `[SETUP-...]` tags in project `user.js`, they give an overall idea of disabled features or important change in behavior.
If your email account(s) is/are already setup, you should be OK (as "accounts auto-configuration" gets disabled by default if you don't override it).

---

Bye, I/we stay around to help/guide you if needed :wave:

**Reply by terry9873** [reply]
Thanks very much. Whilst I appreciate the answers, it does appear it's above my pay grade :( 

I don't use Firefox browser. So much of the help pages refer to Firefox, I wonder if I can even use it if I don't use Firefox. 
I was hoping this was something I could just 'install' and start getting at least some advancement in privacy in Thunderbird. Not a complaint, the work that's gone into this is seriously impressive, by the looks of it. I just simply don't have hours spare to read all these guides, and that's NOT laziness, i literaly don't have the time and even if I did, having skimmed it, I'd probably have 10 questions for every section as the terminology is 'above my pay grade' :( (Also several links go to pages with light text on dark background, ten seconds looking at dark theme pages send me to be for a week with a blinding migraine)

Thanks anyway, I will bookmark this to come back to some day when I understand github and more about tech stuff

**Comment by HorlogeSkynet** [comment]
> I don't use Firefox browser. So much of the help pages refer to Firefox, I wonder if I can even use it if I don't use Firefox.

Of course you can, I was just pointing their wiki out as it is more complete, and this project is inspired by it.

> I was hoping this was something I could just 'install' and start getting at least some advancement in privacy in Thunderbird.

You actually can, simply download `user.js` and copy it to your Thunderbird profile (location may vary in function of OS), restart Thunderbird and you're all set.
You can (should) make a backup of your profile beforehand "just in case".

> I'd probably have 10 questions for every section as the terminology is 'above my pay grade' :(

You can always do this one step at a time and prepare a batch of questions one will be happy to answer when they could.

> Thanks anyway, I will bookmark this to come back to some day when I understand github and more about tech stuff

Thanks to you ! Please note that nothing is related to GitHub : it's only a software forge and if you want to avoid it, this repository is mirrored [here](https://git.forestier.app/HorlogeSkynet/thunderbird-user.js).
Bye :wave:
